### ISSUE TITLE
`Unable to records bytes produced to topic` errors logged in repository meta analyzer

### ISSUE THREAD

**Issue by nscuro** [DependencyTrack/hyades#285]
Whenever `repository-meta-analyzer` produces a result, Kafka Streams logs an error like the following:

```
2023-01-30 12:04:38,195 ERROR [org.apa.kaf.str.pro.int.RecordCollectorImpl] (kafka-producer-network-thread | hyades-repository-meta-analyzer-7308f986-fa3c-43a0-a365-5abc59f77ea9-StreamThread-3-producer) stream-thread [hyades-repository-meta-analyzer-7308f986-fa3c-43a0-a365-5abc59f77ea9-StreamThread-3] task [0_2] Unable to records bytes produced to topic dtrack.repo-meta-analysis.result by sink node produce_npm_result_to_dtrack.repo-meta-analysis.result_topic as the node is not recognized.
Known sink nodes are [produce_empty_result_to_dtrack.repo-meta-analysis.result_topic].
```

Records are produced as expected, so this does not appear to be a show stopper. We should investigate on this anyway and try to resolve it.

**Comment by nscuro** [comment]
This may have been fixed automatically by updating `kafka-streams` to `3.3.2` (included in Quarkus `2.16.1.Final`): https://github.com/apache/kafka/pull/12836

**Comment by nscuro** [comment]
Resolved by upgrade to Quarkus `>=2.16.1.Final`.
